shared/src/payload/primitives.rs

```diff
@@ -38,6 +46,19 @@ impl PriceRatioRaw {
     }
 }
 
+pub struct ConvertU64ToF64 {
```

Comment:
This is a pointless struct. It doesn't add anything you can't do with stdlib - delete it.